src/routes/Homepage/Homepage.tsx

```diff
@@ -31,6 +33,16 @@ export const Homepage = withTranslation()(() => {
     dispatch(Actions.signOut());
   };
 
+  useEffect(() => {
+    const boardDeleted = new URLSearchParams(window.location.search).get("boardDeleted");
+
+    if (boardDeleted) {
+      Toast.error({
```

Comment:
This should be an info toast and not an error toast I guess.

Reply:
Now that you say that, I think it should be the new deletion toast from #3121, shouldn't it?

Reply:
No, it's a simple information for the participants that the board they have been on has been deleted. 

The deletion toast would be for the moderator that has deleted the board.